src/transformState.js

```diff
@@ -46,7 +46,29 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const i of transforms) {
```

Comment:
i - итератор, в этом случае это 
```suggestion
  for (const transform of transforms) {
```
```suggestion
  for (const action of transforms) {
```